resources/integration_test/tests/query.yaml

```
@@ -0,0 +1,197 @@
+tests:
+  - description: select_star_from_grades_ordered
+    db: integration_test
+    sql: SELECT * FROM grades ORDER BY _id
+    ordered: true
+    skip_reason: SQL-659
+
+  - description: select_star_from_class_unordered
```

```suggestion
  - description: select_star_unordered
```
Let's name based on behavior instead of specific test features (in this case, that just means dropping the collection name)